coin.cfg.cmake.in

```diff
@@ -7,18 +7,18 @@ cflags="@COIN_EXTRA_CFLAGS@"
 cppflags="@COIN_EXTRA_CPPFLAGS@"
 cxxflags="@COIN_EXTRA_CXXFLAGS@"
 ldflags="@COIN_EXTRA_LDFLAGS@"
-libs="@COIN_EXTRA_LIBS@"
+libs="@CMAKE_INSTALL_FULL_LIBDIR@"
```

Comment:
The intention of this variable is to hold a list of libs not the installation directory of the Coin3D libraries.